stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/models/ProgressRequestBody.kt

```diff
@@ -16,7 +14,6 @@ internal class ProgressRequestBody(
     private val callback: ProgressCallback
 ) : RequestBody() {
 
-    private val handler = Handler(Looper.getMainLooper())
```

Comment:
How come we don't need this anymore?

Reply:
Because we are not updating progress directly on UI. As the progress is updating in the attachment model and that is observed on the UI thread in ViewModel.

Reply:
Looking at the code that calls into `StreamFileUploader`, we could have existing clients who call `ChatClient#sendFile` and pass in a progress callback to it. If they previously relied on this being called on the UI thread, we'd now break their code.

Reply:
Added a warning for this in Changelog